imageio/config/extensions.py

```diff
@@ -95,7 +95,13 @@ def reset(self):
     FileExtension(
         name="Bitmap",
         extension=".bmp",
-        priority=["pillow", "BMP-PIL", "BMP-FI", "ITK", "pyav"],
+        priority=["pillow", "BMP-PIL", "BMP-FI", "ITK", "pyav", "opencv"],
+        external_link="https://en.wikipedia.org/wiki/BMP_file_format",
+    ),
+    FileExtension(
+        name="Device-Independent Bitmap",
+        extension=".dip",
+        priority=["opencv"],
```

Comment:
Out of curiosity, why are there custom type definitions for some specific formats?

Reply:
Hm? You mean why I added extensions like `.dip`? OpenCV specifies the formats it supports and accompanying extensions in its docs ([here](https://docs.opencv.org/4.x/d4/da8/group__imgcodecs.html#ga288b8b3da0892bd651fce07b3bbd3a56)). I simply took the information from there and copied it here. I figured that if we say "we support what openCV supports" then users would expect formats that work under plain openCV to be equally recognized in ImageIO.

Reply:
Woops, sorry, I added this comment to the wrong lines :)